src/main/java/mate/project/mapper/UserMapper.java

```diff
@@ -0,0 +1,16 @@
+package mate.project.mapper;
+
+import mate.project.config.MapperConfig;
+import mate.project.dto.UserRegistrationRequestDto;
+import mate.project.dto.UserResponseDto;
+import mate.project.model.User;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
```

Comment:
The @Mapper annotation should include componentModel = 'spring' for the UserMapper to be recognized as a Spring bean.